src/server/api/routes/favorites.router.js

```diff
@@ -0,0 +1,104 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const favoritesController = require('../controllers/favorites.controller');
+
+/**
+ * @swagger
+ * /favorites/add:
+ *  post:
+ *    tags:
+ *    - Favorites
+ *    summary: Add a product to favorites for a user
+ *    description:
+ *      Will add a product to favorites for a user.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: favorite
+ *        description: The product to add in favorites.
+ *        schema:
+ *          type: object
+ *          required:
+ *            - user_id
+ *            - product_id
+ *
+ *
+ *          properties:
+ *            user_id:
+ *              type: integer
+ *            product_id:
+ *              type: integer
+ *
+ *
+ *
+ *    responses:
+ *      201:
+ *        description: added to favorites
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/add', (req, res) => {
+  favoritesController
+    .addToFavorites(req.body)
+    .then((result) => res.json(result))
+    .catch((error) => {
+      console.log(error);
+
+      res.status(400).send('Bad request').end();
+    });
+});
+
+/**
+ * @swagger
+ * /favorites/delete:
+ *  post:
+ *    tags:
+ *    - Favorites
+ *    summary: Delete a product from favorites from a user
+ *    description:
+ *      Will delete a favorite product for a user.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
```

Comment:
parameters are not correct, Please refer to modules router for idea

Reply:
pls review @Divyajg